scripts/install_python_dev_dependencies.py

```diff
@@ -82,16 +85,23 @@ def install_installation_tools() -> None:
 
 
 def install_dev_dependencies() -> None:
-    """Install dev dependencies from
-    COMPILED_REQUIREMENTS_DEV_FILE_PATH.
-    """
+    """Install dev dependencies from COMPILED_REQUIREMENTS_DEV_FILE_PATH."""
     subprocess.run(
         ['pip-sync', COMPILED_REQUIREMENTS_DEV_FILE_PATH],
         check=True,
         encoding='utf-8',
     )
 
 
+def uninstall_dev_dependencies() -> None:
+    """Uninstall dev dependencies from COMPILED_REQUIREMENTS_DEV_FILE_PATH."""
+    subprocess.run(
+        ['pip', 'uninstall', '-r', COMPILED_REQUIREMENTS_DEV_FILE_PATH, '-y'],
+        check=True,
+        encoding='utf-8',
+    )
```

Comment:
After this, should we double-check that no Python packages (besides the installation tools if we want those) are present? This could help in case the user installs a package manually.